Hints.py

```diff
@@ -972,6 +972,51 @@ def get_junk_hint(spoiler, world, checked):
+    for location in world.get_filled_locations():
+        if (HintArea.at(location).text(world.settings.clearer_hints) not in top_level_locations 
+            and (HintArea.at(location).text(world.settings.clearer_hints) + ' Important Check') not in checked
+            and "Pocket" not in HintArea.at(location).text(world.settings.clearer_hints)):
+            top_level_locations.append(HintArea.at(location).text(world.settings.clearer_hints))
+    hintLoc = random.choice(top_level_locations)
+    item_count = 0
+    for location in world.get_filled_locations():
+        region = HintArea.at(location).text(world.settings.clearer_hints)
+        if region == hintLoc:
+            if (location.item.majoritem
+                and not location.item.name == 'Triforce Piece'
+                and not location.item.name == 'Deliver Letter'
+                and not ((location.name == 'Song from Impa' or location.item.name == 'Zeldas Letter') and world.settings.skip_child_zelda)
+                and not(location.item.name == 'Kokiri Sword' and not world.settings.shuffle_kokiri_sword
+                or location.item.name == 'Giants Knife' and not world.settings.shuffle_medigoron_carpet_salesman
+                or location.item.name == 'Gerudo Membership Card' and not world.settings.shuffle_gerudo_card
+                or location.item.name == 'Ocarina' and not world.settings.shuffle_ocarinas
+                or 'Bean' in location.item.name and not world.settings.shuffle_beans
+                or location.item.name == 'Weird Egg' and not world.settings.shuffle_weird_egg)
+                or (location.item.type == 'SmallKey' and not (world.settings.shuffle_smallkeys == 'dungeon' or world.settings.shuffle_smallkeys == 'vanilla'))
+                or (location.item.type == 'HideoutSmallKey' and not world.settings.shuffle_hideoutkeys == 'vanilla')
+                or (location.item.type == 'BossKey' and not (world.settings.shuffle_bosskeys == 'dungeon' or world.settings.shuffle_bosskeys == 'vanilla'))
+                or (location.item.type == 'GanonBossKey' and not (world.settings.shuffle_ganon_bosskey == 'vanilla' 
+                or world.settings.shuffle_ganon_bosskey == 'dungeon' or world.settings.shuffle_ganon_bosskey == 'on_lacs'
+                or world.settings.shuffle_ganon_bosskey == 'stones' or world.settings.shuffle_ganon_bosskey == 'medallions'
+                or world.settings.shuffle_ganon_bosskey == 'dungeons' or world.settings.shuffle_ganon_bosskey == 'tokens'))):
+                item_count = item_count + 1
+
+    checked.add(hintLoc + ' Important Check')
+
+    if item_count == 0:
+        numcolor = 'Red'
+    elif item_count == 1:
+        numcolor = 'Pink'
+    elif item_count == 2:
+        numcolor = 'Yellow'
+    elif item_count == 3:
+        numcolor = 'Light Blue'
+    else:
+        numcolor = 'Green'
+    return (GossipText('#%s# has #%d# major items.' % (hintLoc, item_count), [numcolor, 'Green']), None)
```

Comment:
It's a minor thing but, could you change it so that the s in "items" is removed when there's only 1?

Reply:
Updated